src/tactic/converter/apply_congr.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Lucas Allen, Scott Morrison
+-/
+
+import tactic.interactive
+import tactic.converter.interactive
+
```

Comment:
Missing module doc. There's not much in this file, so it can be short, but it should be there.

Reply:
Done.